provider/go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/pulumi/pulumi-newrelic/provider/v4
 go 1.16
 
 require (
-	github.com/hashicorp/terraform-plugin-sdk v1.17.2 // indirect
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.15.0 // indirect
 	github.com/newrelic/terraform-provider-newrelic/v2 v2.2.1
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.21.0
-	github.com/pulumi/pulumi/sdk/v3 v3.30.0
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.23.0
+	github.com/pulumi/pulumi/sdk/v3 v3.32.1
 )
 
 replace (
 	github.com/hashicorp/go-getter v1.5.0 => github.com/hashicorp/go-getter v1.4.0
-	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20210629210550-59d24255d71f
-	github.com/newrelic/terraform-provider-newrelic/v2 => github.com/pulumi/terraform-provider-newrelic/v2 v2.19.1-0.20220422210324-2b739dd72124
+	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220505215311-795430389fa7
```

Comment:
Be careful about this version matching upstream since we've seen some bugs reported with providers without adequate acceptance test coverage.